packages/bidirection/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@material/bidirection",
```

Comment:
If this is for MDC React only, it should be `react-` prefixed.

If this is applicable to JS in general, we should consider adding it to `mdc-rtl` instead in MDC Web... but we should also look at current implementations in MDC Web (e.g. I'm pretty sure several adapters have `isRtl` functions)